cabal-ghc90.project

```diff
@@ -27,8 +27,6 @@ packages:
          ./plugins/hls-call-hierarchy-plugin
          ./plugins/hls-alternate-number-format-plugin
 
-with-compiler: ghc-9.0.1
```

Comment:
I'm assuming this didn't matter? We won't have another GHC in scope or anything? (I don't think we will, but just checking)

Reply:
the line only saved you `-w ghc-9.0.1`. Now the file should work for both ghc 9.0.1 and 9.0.2 and that line would make 9.0.2 in PATH (like we do in ci) be ignored